.github/workflows/ci_ut_ext_blob_workflow.yml

```diff
@@ -4,12 +4,10 @@ on:
     push:
       branches: [ dev, master, main, release/* ]
       paths:
-        - 'azure-functions-extension-base/**'
         - 'azure-functions-extension-blob/**'
     pull_request:
         branches: [ dev, master, main, release/* ]
         paths:
-          - 'azure-functions-extension-base/**'
```

Comment:
Why is this removed?

Reply:
Can you update the description of this PR. Not sure whats happening here

Reply:
updated, as we discussed if we dont install editable base package in fastapi or blob pipeline for testing, there is no need to trigger blob or fastapi if base repo changes.